app/stores/AdaRedemptionStore.js

```diff
@@ -30,6 +30,7 @@ export default class AdaRedemptionStore extends Store {
   @observable amountRedeemed: number = 0;
   @observable showAdaRedemptionSuccessMessage: boolean = false;
   @observable redeemAdaRequest: Request<Wallet> = new Request(this.api.redeemAda);
+  @observable redeemAdaSubmitted: boolean = false;
```

Comment:
@DominikGuzei I have refactored the code so that this variable is not needed at all. Please review again ;)